sycl/test/basic_tests/image/image_write.cpp

```diff
@@ -0,0 +1,282 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+
+/**
+ * Tests clamp_to_edge addressing mode with nearest and linear filtering modes
+ * on a 4x4 image.
+ *
+ * Expected addressing mode and filtering results are given by the algorithm in
+ * the OpenCL 1.2 specification, Section 8. Image Addressing and Filtering
+ *
+ * https://www.khronos.org/registry/OpenCL/specs/opencl-1.2.pdf#page=329
+ *
+ * Confirmed to pass with -fsycl-targets=spir64-unknown-linux-sycldevice
+ *
+ * Enable sampled test with -DSAMPLED_TEST
```

Comment:
-DUNSAMPLED_TEST and -DSAMPLED_TEST are not used in test code. Cane we remove corresponding comments?